docs/experiments-msmarco-passage.md

```diff
@@ -0,0 +1,160 @@
+# PyGaggle: Neural Ranking Baselines on [MS MARCO Passage Retrieval](https://github.com/microsoft/MSMARCO-Passage-Ranking)
+
+This page contains instructions for running various neural reranking baselines on the MS MARCO *passage* ranking task. 
+Note that there is also a separate [MS MARCO *document* ranking task](experiments-msmarco-doc.md).
+
+Prior to running this, we suggest looking at our first-stage [BM25 ranking instructions](https://github.com/castorini/anserini/blob/master/docs/experiments-msmarco-passage.md).
+We rerank the BM25 run files that contain ~ 1000 passages per query using both monoBERT and monoT5.
+
+Keeping computational resources in mind, our instructions primarily focus on a 105 query subset of the MS MARCO dev set. 
```

Comment:
```suggestion
Since it can take many hours to run these models on all 6980 queries from the MS MARCO dev set, we will use instead a subset of 105 queries randomly sampled from that set. 
```